dask_kubernetes/conftest.py

```diff
@@ -1,15 +1,26 @@
 import pytest
 
+from glob import glob
+import pathlib
 import os
 import subprocess
 
+from kopf.testing import KopfRunner
+
 from dask_kubernetes.common.utils import check_dependency
 
+DIR = pathlib.Path(__file__).parent.absolute()
+
 check_dependency("helm")
 check_dependency("kubectl")
 check_dependency("docker")
 
 
+@pytest.fixture()
+async def kopf_runner(k8s_cluster):
```

Comment:
why is this an `async def` fixture -

https://github.com/nolar/kopf/blob/ab53ace82e62a6fa709bf5a580007eac1273ac34/kopf/_kits/runner.py#L123

kopf_runner creates its own eventloop in its own thread

Reply:
This is likely a typo :)